lms/djangoapps/discussion/static/discussion/templates/fake-breadcrumbs.underscore

```diff
@@ -0,0 +1,9 @@
+<h6 class="hd-6 fake-breadcrumbs">
```

Comment:
Nit: our convention is four spaces for HTML (even though we don't always follow it).